frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -77,36 +83,52 @@ const getColumns = () => {
   ];
 };
 
+const getNodeTaints = (node: NodeKind) => {
+  return (node.spec && node.spec.taints) || [];
```

Comment:
This helper is not needed imo. Better to use `const taints: Taint[] = node.spec.taints || []` inline. `spec` will not be null.

You're losing type safety because you haven't declared the return type and not all code paths return `Taint[]`.